ideascube/library/migrations/0008_library_stock.py

```diff
@@ -163,7 +163,7 @@ class Migration(migrations.Migration):
             },
             bases=('monitoring.specimen', models.Model),
         ),
-        migrations.RunPython(migrate_data),
+        migrations.RunPython(migrate_data, hints={'using': 'default'}),
```

Comment:
A migration have to be reversible to allow migration tests to test this migration or migration before it.
As this migration already exists and we do not test it or one of this dependencies we do not need to make it reversible.